### PR TITLE
Upload component

### PR DESCRIPTION
**Pull request by amitmishrg**

### CODE REVIEW

File: src/js/components/UploadFile/ValidateTable.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          />
+        </Box>
+      </Box>
+    </Box>
+  );
+};
+
+ValidateTable.propTypes = {
+  tableData: PropTypes.shape({
+    headers: PropTypes.shape([]),
```
Comment (sabarnix) [inline src/js/components/UploadFile/ValidateTable.js:108]: shape not defined

**Comment by sabarnix** [comment]
Documentation and typescript missing

**Comment by sabarnix** [comment]
@amitmishrg storybook build is failing on netlify

File: src/js/components/UploadFile/ValidateTable.js (hunk 1 above)
Comment (amitmishrg) [reply src/js/components/UploadFile/ValidateTable.js]: Fixed

**Pull request closed without merging**